BIDS-Dependencies-List.md

```diff
@@ -0,0 +1,31 @@
+# BIDS Dependencies List
+The following is a list of **options** available for working with BIDS
+
+You **do not** need all these programs to work with BIDS. Choose whichever ones you are most comfortable with
+
+### MATLAB 
+***Information***: https://www.mathworks.com/products/matlab.html 
+
+***Free, Open-source equivalent***: Octave (https://www.gnu.org/software/octave/)
+
+**JSONio** (package required to read and write JSON files)
+
+		Install Link: https://github.com/gllmflndn/JSONio 
+
+### Python
```

Comment:
Do Python & R folks need a json editor (as with the matlab users)?

Maybe mention that they'll need a text editor?

Reply:
+1 to text editor - otherwise this becomes a forking paths problem (e.g. there's a JSON editor in some IDEs but not others...we should just mention this general fact and suggest they use the text editor of their choice, maybe linking to Atom or some other common one)